$NetBSD: patch-ag,v 1.2 2009/02/21 18:54:41 gdt Exp $

From Giles Lean.  Discards characters that may be waiting in the tty
on open to get a fresh start.  Upstream said would be applied on
2009-02-21.

--- jeeps/gpsserial.c.orig	2008-09-18 16:16:14.000000000 -0400
+++ jeeps/gpsserial.c
@@ -281,7 +281,7 @@ int32 GPS_Serial_Open(gpsdevh *dh, const
     tty.c_cc[VMIN] = 1;
     tty.c_cc[VTIME] = 0;
 
-    if(tcsetattr(psd->fd,TCSANOW,&tty)==-1)
+    if(tcsetattr(psd->fd,TCSANOW|TCSAFLUSH,&tty)==-1)
     {
 	GPS_Serial_Error("SERIAL: tcsetattr error");
 	return 0;
